### DISCUSSION TITLE
Learning to use graphia; tried GraphML-Attributes Example but it ignores the attributes

### DISCUSSION THREAD

**Discussion by mdalypgh** [graphia-app/graphia#76] [General]
I am working with a relatively small data set but I cannot get attributes to work. So I just loaded the example at http://graphml.graphdrawing.org/primer/graphml-primer.html#AttributesExample and it doesn't set the attributes either. The attached screenshot is what gets generated. Did I miss installing something?

![Capture](https://github.com/graphia-app/graphia/assets/67755800/2f952399-f64c-4bee-a089-822437c38423)

**Comment by timangus** [comment]
That looks normal to me. Perhaps you're expecting the attribute value to be visualised in some way? You'll need to click *Add Visualisation* for that...

**Reply by mdalypgh** [reply]
I was expecting the app to apply the instructions found in the file as the file was read. That's how I've implemented software in the past. Then, the user could filter the instructions out if necessary. But, thanks, now I understand the strategy you've used. I just tested the _add visualization_ and I see how it works now. Again, thanks for the quick reply.
